### PR TITLE
Dealer get card function

### PR DESCRIPTION
**Pull request by php-121**
simple function determines if the house choose to get card(s) based on player's known hand

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -161,3 +175,34 @@ def dealCards(self, numCards: int, players: "list[Player]"):
...
+
+# simple respond for dealer in single player with known hands 
+def dealerHand(dealer, house, player):
+  houseHand = house.handSum()
+  playerHand = player.handSum()
+  if playerHand == 21 and len(player.hand) == 2: #if blackjack then player auto win
+    return 1   
+  if houseHand == 21 and len(house.hand) == 2: #if blackjack then house auto win
+    return -1 
+  while houseHand < 16: #force hit when below 16
```
Comment (grant-warkentin) [inline source/CardGames.py:187]: The house should hit while houseHand < 17

Comment (php-121) [reply source/CardGames.py]: fixed

**Review by grant-warkentin — APPROVED** [review]

File: source/CardGames.py [hunk 2]
```diff
@@ -161,3 +175,35 @@ def dealCards(self, numCards: int, players: "list[Player]"):
...
+  if playerHand == 21 and len(player.hand) == 2: #if blackjack then player auto win
+    return 1   
+  if houseHand == 21 and len(house.hand) == 2: #if blackjack then house auto win
+    return -1 
+  while houseHand < 17: #force hit when below 17
+    dealer.dealCards(1, [house])
+    houseHand = house.handSum()
+
+  if playerHand <= 21: 
+    while playerHand >= houseHand: # keep hitting until house hand value is bigger than player hand, unless houseHand reach 21
```
Comment (grant-warkentin) [inline source/CardGames.py:192]: The house does not respond to the player. The house operates on a strict algorithm where it must hit if their hand is <17 else they stand if their handsum is >= 17.

File: source/CardGames.py [hunk 3]
```diff
@@ -161,3 +175,35 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
+
+# simple respond for dealer in single player with known hands 
+def dealerHand(dealer, house, player):
+  houseHand = house.handSum()
+  playerHand = player.handSum()
+  if playerHand == 21 and len(player.hand) == 2: #if blackjack then player auto win
+    return 1
```
Comment (grant-warkentin) [inline source/CardGames.py:184]: If the player and the house both have blackjack it is a push.

**Review by grant-warkentin — APPROVED** [review]
Besides these minor changes, everything looks good.

**Review by TommyMedina — APPROVED** [review]
according to: https://en.wikipedia.org/wiki/Blackjack#Rules_of_play_at_casinos if a player has a blackjack and the house also has a blackjack the hands tie.

File: source/CardGames.py (hunk 3 above)
Comment (php-121) [reply source/CardGames.py:184]: Gotcha, so it's a tie

**Pull request merged**